config/pre-install/v0.18.0/pingsource.yaml

```diff
@@ -0,0 +1,208 @@
+                          current state.'
+                      type: array
+                      items:
+                          type: object
+                          required:
+                            - type
+                            - status
+                          properties:
+                              lastTransitionTime:
+                                  description: 'LastTransitionTime is the last time the condition
+                                      transitioned from one status to another. We use VolatileTime
+                                      in place of metav1.Time to exclude this from creating
+                                      equality.Semantic differences (all other things held
+                                      constant).'
+                                  type: string
+                              message:
+                                  description: 'A human readable message indicating details
+                                      about the transition.'
+                                  type: string
+                              reason:
+                                  description: 'The reason for the condition''s last transition.'
+                                  type: string
+                              severity:
+                                  description: 'Severity with which to treat failures of
+                                      this type of condition. When this is not specified,
+                                      it defaults to Error.'
+                                  type: string
+                              status:
+                                  description: 'Status of the condition, one of True, False,
+                                      Unknown.'
+                                  type: string
+                              type:
+                                  description: 'Type of condition.'
+                                  type: string
+                  observedGeneration:
+                      description: 'ObservedGeneration is the "Generation" of the Service
+                          that was last processed by the controller.'
+                      type: integer
+                      format: int64
+                  sinkUri:
+                      description: 'SinkURI is the current active sink URI that has been
+                          configured for the Source.'
+                      type: string
+    additionalPrinterColumns:
+    - name: Ready
+      type: string
+      jsonPath: ".status.conditions[?(@.type=='Ready')].status"
+    - name: Reason
+      type: string
+      jsonPath: ".status.conditions[?(@.type=='Ready')].reason"
+    - name: Sink
+      type: string
+      jsonPath: .status.sinkUri
+    - name: Age
+      type: date
+      jsonPath: .metadata.creationTimestamp
+  - << : *version
+    name: v1alpha2
+    served: false
```

Comment:
to be on the safe side I would mark it as served (served: true).

Reply:
+1.

I think the CRD here should be exactly the same as the CRD in `config/core/resources/.....`